packages/swapper/src/swappers/zrx/utils/helpers/helpers.ts

```diff
@@ -53,27 +58,34 @@ export const isNativeEvmAsset = (assetId: AssetId): boolean => {
       return assetId === avalancheAssetId
     case KnownChainIds.OptimismMainnet:
       return assetId === optimismAssetId
+    case KnownChainIds.BnbSmartChainMainnet:
+      return assetId === bscAssetId
     default:
       return false
   }
 }
 
-export const getUsdRate = async (asset: Asset): Promise<string> => {
-  const { assetReference: erc20Address, assetNamespace } = fromAssetId(asset.assetId)
-  const { symbol } = asset
+export const assetToToken = (asset: Asset): string => {
+  const { assetReference, assetNamespace } = fromAssetId(asset.assetId)
+  return assetNamespace === 'slip44' ? asset.symbol : assetReference
+}
 
+export const getUsdRate = async (sellAsset: Asset): Promise<string> => {
   try {
-    const USDC_CONTRACT_ADDRESS = usdcContractFromChainId(asset.chainId)
-    if (erc20Address?.toLowerCase() === USDC_CONTRACT_ADDRESS) return '1' // Will break if comparing against usdc
-    const baseUrl = baseUrlFromChainId(asset.chainId)
+    const usdcToken = usdcTokenFromChainId(sellAsset.chainId)
+    const sellToken = fromAssetId(sellAsset.assetId).assetReference
+
+    if (sellToken === usdcToken) return '1' // Will break if comparing against usdc
+
+    const baseUrl = baseUrlFromChainId(sellAsset.chainId)
     const zrxService = zrxServiceFactory(baseUrl)
     const rateResponse: AxiosResponse<ZrxPriceResponse> = await zrxService.get<ZrxPriceResponse>(
       '/swap/v1/price',
       {
         params: {
-          buyToken: USDC_CONTRACT_ADDRESS,
+          buyToken: usdcToken,
```

Comment:
Ditto, the previous const carried the notion of this being an `account` / `address`, now this doesn't anymore :(

Reply:
https://github.com/shapeshift/lib/pull/1219/commits/8aadfe5e8efa957692f6bf8b9eafc0b8ece17d12